DDP-5744 Refresh model (activity) after wizard steps navigation

alextanlan opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/broadinstitute/ddp-angular/492)
<!-- Reviewable:end -->

EdPokhvalin committed:
DDP-5747 Move the composite answer component to a separate directory, make additional files for the template and styles

EdPokhvalin committed:
DDP-5747 Override styles for the close button in the composite answer component

alextanlan committed:
Refresh model (activity) after wizard steps navigation

EdPokhvalin reviewed (dismissed):
Everything seems ok

> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/broadinstitute/ddp-angular/492#-:-MXBzFC-BntCgbX6aBRn:buxwa3)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @fat23cat and @MocanaAtBroad)



<!-- Sent from Reviewable.io -->

alextanlan reviewed (commented):
Not yet. It's a draft for now because some unpredictable issue appears. I am on it.

> *[Reviewable](https://reviewable.io/reviews/broadinstitute/ddp-angular/492#-:-MXC1CqzAIIjDhW5jraN:b-jq0h1u)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @fat23cat and @MocanaAtBroad)



<!-- Sent from Reviewable.io -->

alextanlan committed:
Revert "Refresh model (activity) after wizard steps navigation"

This reverts commit b03b77ca
because it was a wrong decision

alextanlan committed:
Change activityBlock in order to fix the bug

  now if we change (add/remove) a childInstance the main activity changed properly
  during wizard navigation

alextanlan committed:
Mutate childInstances in main activity

 in order changes to be be saved during wizard navigation
